src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/Commands/Synchronize-AzureASInstance.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using System.Linq;
+using System.Management.Automation;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.AnalysisServices.Dataplane.Models;
+using Microsoft.Azure.Commands.AnalysisServices.Dataplane.Properties;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using System.Net;
+
+namespace Microsoft.Azure.Commands.AnalysisServices.Dataplane
+{
+    /// <summary>
+    /// Cmdlet to log into an Analysis Services environment
+    /// </summary>
+    [Cmdlet("Synchronize", "AzureAnalysisServicesInstance", SupportsShouldProcess = true)]
```

Comment:
Please add Unit Tests  - D:\github\azure-powershell\src\ResourceManager\AnalysisServices\Commands.AnalysisServices.Test\InMemoryTests\DataPlaneCommandTests.cs. 

Scenario test should be straig forward as well.